scripts/webhook-deploy.sh

```diff
@@ -36,6 +36,14 @@ while [[ $# -gt 0 ]]; do
             namespace="$2"
             shift
             ;;
+	--uid)
+            uid="$2"
```

Comment:
would it be good to remind user to update Dockerfile too if the default is overridden?

Reply:
I tend to remove this option and use hardcoded uid and gid. It seems to create more problems than solves.
@mythi wdyt?

Reply:
+1

Reply:
done